app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -611,4 +614,11 @@ class StateFragmentPresenter @Inject constructor(
   }
 
   private fun isAudioShowing(): Boolean = viewModel.isAudioBarVisible.get()!!
+
+  fun updateSubmitButton(pendingAnswerError: String?) {
+    if (pendingAnswerError != null)
```

Comment:
Here & elsewhere: all statement blocks should have braces, even for single statements (unless they fit on one line), per https://developer.android.com/kotlin/style-guide#formatting. I didn't know about this earlier, so some of our other code may not exactly follow that yet, but new code should.

Reply:
Done.